testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -10754,6 +10757,17 @@ private void testCurrentDateFunc(Pair<String, Hook.Closeable> pair) {
     f0.forEachLibrary(libraries, consumer);
   }
 
+  @Test void differentTypeValues() {
```

Comment:
Does this test belong here? It isn't testing an operator, or use the fixture.

Could it be a Quidem test?

Reply:
I will move it.
I don't know if I can create the same environment in Quidem. A simple attempt to reproduce this bug with standard settings failed.